OIPA/api/export/tests/test_full_activity.py

```diff
@@ -208,6 +210,7 @@ def test_create_activity(self):
 
 
         xml = iati_activities(
+                ET.Comment(settings.EXPORT_COMMENT),
```

Comment:
It is for adding a comment in the XML, so in the case of IS a comment indicating the XML was exported from IS. I added the string to base_settings.py